### PR TITLE
added startAngle property

### PR DESCRIPTION
**Pull request by whamtet**

### CODE REVIEW

File: src/Sectors/index.js [hunk 1]
```diff
@@ -10,25 +10,26 @@ const Sectors = ({
...
   const total = data.reduce((prev, current) => current.value + prev, 0)
-  let angleStart = 0
-  let angleEnd = 0
+  let angleStart = startAngle
+  let angleEnd = startAngle
   return total > 0 ? (
     <g>
       {data.map((d, i) => {
         const isLarge = d.value / total > 0.5
-        const angle = (360 * d.value) / total
```
Comment (whamtet) [inline src/Sectors/index.js:22]: This change does not exist in the pull request, but it is showing due to a bug in Github.

File: src/Sectors/index.js [hunk 2]
```diff
@@ -10,25 +10,26 @@ const Sectors = ({
...
+        const angle = 360 * d.value / total
         const radius = center + (d.expanded ? expandSize : 0) - strokeWidth / 2
 
         angleStart = angleEnd
         angleEnd = angleStart + angle
 
-        const x1 = center + radius * Math.cos((Math.PI * angleStart) / 180)
```
Comment (whamtet) [inline src/Sectors/index.js:28]: This change does not exist in the pull request, but it is showing due to a bug in Github.

**Review by cedricdelpoux — APPROVED** [review]

**Comment by cedricdelpoux** [comment]
Thank you @whamtet

**Comment by whamtet** [comment]
Thanks for merging!

**Pull request merged**